src/internal/cache-client.ts

```diff
@@ -156,11 +156,12 @@ export class CacheClient {
   private async sendDelete(
     cacheName: string,
     key: Uint8Array
-  ): Promise<DeleteResponse> {
+  ): Promise<CacheDelete.Response> {
     const request = new cache.cache_client._DeleteRequest({
       cache_key: key,
     });
     const metadata = this.createMetadata(cacheName);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     return await new Promise((resolve, reject) => {
```

Comment:
nit: can remove `reject` and then you don't need to disable any eslint rules here

Reply:
Nice, thanks.